pkgs/games/btanks/default.nix

```diff
@@ -14,19 +15,24 @@ stdenv.mkDerivation rec {
 
   NIX_CFLAGS_COMPILE = "-I${SDL_image}/include/SDL";
 
+  prePatch = ''
+    substituteInPlace ./engine/SConscript --replace "lua5.1" "lua" \
+          --replace "lua5.0" "lua"
+  '';
+
   patches = [
     (fetchpatch {
-      url = "https://sources.debian.org/data/main/b/btanks/0.9.8083-7/debian/patches/gcc-4.7.patch";
+      url = "https://salsa.debian.org/games-team/btanks/raw/master/debian/patches/gcc-4.7.patch";
       sha256 = "1dxlk1xh69gj10sqcsyckiakb8an3h41206wby4z44mpmvxc7pi4";
     })
     (fetchpatch {
-      url = "https://sources.debian.org/data/main/b/btanks/0.9.8083-7/debian/patches/pow10f.patch";
+      url = "https://salsa.debian.org/games-team/btanks/raw/master/debian/patches/pow10f.patch";
       sha256 = "1h45790v2dpdbccfn6lwfgl8782q54i14cz9gpipkaghcka4y0g9";
     })
   ];
 
   meta = with stdenv.lib; {
-    homepage = https://sourceforge.net/projects/btanks/;
+    homepage = "https://sourceforge.net/projects/btanks/";
```

Comment:
Why would we want this to be a string?

Reply:
there were some talks of not recognizing urls from nix anymore so some people have been preemptively converting these to strings.

Reply:
I think that's silly; it's easy to do a treewide `sed` if that happens. And if it doesn't, it's a waste of time.

Do you have a link to the discussion about removing URL support? I didn't hear about it.

Reply:
https://github.com/NixOS/rfcs/pull/45